### ISSUE TITLE
Two Identical Journal Titles in Client App

### ISSUE THREAD

**Issue by nrshapiro** [Psychoanalytic-Electronic-Publishing/OpenPubArchive-Content-Server#192]

**Comment by nrshapiro** [comment]
> As a note, in the KBART these are listed as follows:
> 
> Revista de psicoanálisis (APM) Revista de psicoanálisis (APA)

@davidtuckett @SophieMBennett 

The issue with (APA) on the second is that APA is our code we use all over the place for JAPA, which is why I suggested REVAPA, which is the code we use for REVAPA!

It doesn't make a functional difference, so I can do that, but knowing the correct codes is actual of great benefit to the end-user.  It might not be (a bad idea at some point) to put the codes after all the journals.  Again, this is why I don't think we should use APA.  For now, I'll go with REVAPA, but I can change it if you need.

**Comment by davidtuckett** [comment]
Yes. I have often worried about this. I am not sure there is a an easy solution unless we use words. eg Madrid, Argentina. I think we should use JAPA if we can. Not sure hoqw we ever used APA!
